src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletOperationsTests.cs

```diff
@@ -12,6 +12,7 @@
 using NBitcoin.DataEncoders;
 using Newtonsoft.Json;
 using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
```

Comment:
This file also some commented out tests. Please delete or comment.

Reply:
Restored.